fpdf/fpdf.py

```diff
@@ -862,6 +864,52 @@ def set_line_width(self, width):
         if self.page > 0:
             self._out(f"{width * self.k:.2f} w")
 
+    @contextmanager
+    @check_page
+    def drawing_context(self, dump_render_tree=None):
```

Comment:
Could you please indicate what must be the type of `dump_render_tree` here?
Either through the docstring or as type hint

Reply:
I believe the docstring does provide some basic documentation for this, but the type is a writable text stream (e.g. `sys.stderr` or `io.StringIO`). I'm open to suggestions on improving the docstring for this.

Reply:
I missed it in the docstring, my bad.
It's totally fine as it is for me, just ignore my remark here.